node/CHANGELOG.md

```diff
@@ -30,11 +30,14 @@ All notable changes to this project will be documented in this file.  The format
 * More node modules are now `pub(crate)`.
 * Chain automatically creates a switch block immediately after genesis or an upgrade.
 
+### Deprecated
+* Deprecate the `starting_state_root_hash` field from the REST and JSON-RPC status endpoints.
```

Comment:
It's now _removed_ I suppose.

Reply:
As per discussion, this is apparently a breaking change to remove it so I've reverted.

Reply:
So I'm looking here and it's still in: https://github.com/xcthulhu/casper-node/blob/1ef261249fd8a542cc5607882737665ce055c096/node/src/types/status_feed.rs#L172

Where do you see it missing?

Reply:
Hmm, it was removed when I checked. If it's back, that's fine.